server/cpho/jinja2/indicators/view_indicator.jinja2

```diff
@@ -1,21 +1,32 @@
 {% extends 'base.jinja2' %}
 {% block content %}
-  <div>{{ object.name }}</div>
+  <h2>{{ tdt('Indicator Selected: ')}} {{ object.name }}</h2>
 
-  <div>
-    <a href="{{ url('edit_indicator', args=[object.id]) }}">{{ tdt('Edit indicator') }}</a>
+  <div class="mt-4">
+    <a href="{{ url('edit_indicator', args=[object.id]) }}"
+       class="btn btn-primary">{{ tdt('Edit indicator') }}</a>
   </div>
-
-  <div>
-    <ul>
-      {% for dimension_type in dimension_types %}
-        <li>
-          <a href="{{ url('manage_indicator_data', args=[object.id, dimension_type.id]) }}">{{ tdt('Manage data for ')}} {{ dimension_type.name }}</a>
-        </li>
-      {% endfor %}
-      <li>
-        <a href="{{ url('manage_indicator_data_all', args=[object.id]) }}">{{ tdt('Manage data for ')}} {{ tdt("all") }}</a>
-      </li>
-    </ul>
+  <div class="mt-5">
+ 
+    {% for dimension_type in dimension_types %}
+      <h5>
```

Comment:
I think there are a11y rules around h5 must come after h4, h3s, etc. 
If you just want the style, you can use `class="h5"`